reliability/level2/README.md

```diff
@@ -0,0 +1,15 @@
+# Level 2
+
+## Summary
+
+The `level2` program will send log messages to a local HTTP server at http://localhost:3000/.
```

Comment:
if there are multiple programs, then the initial brief of launching `$ ruby main` might be a bit unclear

Reply:
Right, I should probably add something to make the distinction between the `levelX` (generator) and `main.rb` (solution)